### PR TITLE
Entropy Solidity SDK & usage example 

### PR DESCRIPTION
**Pull request by jayantk**
I wanted to make an example for how to use entropy. In order to do that, I needed to extract out the interface of the entropy contract into an SDK. 

There is still work to do on the example. I've only added the solidity contract for the moment and checked that it builds. I think we also need to generate ABIs in the solidity SDK and then show how to interact with the example contract via typescript. I think that can wait for a future PR though because there's already a decent amount of stuff here.

### CODE REVIEW

File: fortuna/config.yaml [hunk 1]
```diff
@@ -2,3 +2,9 @@ chains:
   optimism-goerli:
     geth_rpc_addr: https://goerli.optimism.io
     contract_addr: 0x28F16Af4D87523910b843a801454AEde5F9B0459
+  avalanche-fuji:
+    geth_rpc_addr: https://api.avax-test.network/ext/bc/C/rpc
+    contract_addr: 0xD42c7a708E74AD19401D907a14146F006c851Ee3
+  eos-evm-testnet:
+    geth_rpc_addr: https://api.testnet.evm.eosnetwork.com/
+    contract_addr: 0xD42c7a708E74AD19401D907a14146F006c851Ee3
```
Comment (jayantk) [inline fortuna/config.yaml:10]: drive-by: deployed on some other networks

File: target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol [hunk 2]
```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache 2
```
Comment (jayantk) [inline target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol:1]: this is copy/paste of the interface from PythRandom.

I've called this IEntropy to match the new name scheme. I'll circle back and rename the other contracts as well (didn't want to intermingle in this one PR).

File: fortuna/config.yaml (hunk 1 above)
Comment (ali-bahjati) [reply fortuna/config.yaml:10]: ToDo: contract manager support

**Review by Reisen — APPROVED** [review]
